doc/nrf/getting_started/modifying.rst

```diff
@@ -198,10 +240,10 @@ The documentation for each :ref:`configuration option <configuration_options>` l
       To locate a specific configuration option, use the **Jump to** field.
 
 
-Changing the configuration permanently
-======================================
+Permanent changes
+-----------------
 
-To configure your application and maintain the configuration when you clean the build directory pristinely, you need to specify the configuration in one of the permanent configuration files.
+To configure your application and maintain the configuration after you clean the build directory pristinely, you need to specify the configuration in one of the permanent configuration files.
```

Comment:
Again, similar comment as above. I just want to make sure that the developers agree this is a commonly used phrase.

Reply:
See my comment above.